scripts/updatechangelog.sh

```diff
@@ -0,0 +1,11 @@
+output="Please refer to each API's \`CHANGELOG.md\` file under the \`packages/\` directory\n\nChangelogs\n-----\n"
+
+SCRIPT_ROOT=$(realpath $(dirname "${BASH_SOURCE[0]}"))
+PACKAGES_URL=https://github.com/googleapis/google-cloud-python/tree/main/packages
+
+for package in $SCRIPT_ROOT/../packages/*/ ; do
+    package_dir=$(basename $package)
+    output+="[$package_dir]("$PACKAGES_URL/$package_dir"/CHANGELOG.md)\n"    
+done
+
+echo -e $output > $SCRIPT_ROOT/../CHANGELOG.md
```

Comment:
So to confirm, looking at https://github.com/googleapis/google-cloud-python/pull/10836/files: we're explicitly and intentionally getting rid of the single repo-wide numbering, yes? Why did we have it before? Was it simply something we always intended to fix and never got to?

I agree what you do in this PR makes more sense, just wondering about context and possible unintended consequences.

Reply:
I'm not aware of how the single repo-wide numbering was configured previously. We haven't used it since we've migrated back to the monorepo.